workloads/templates/workload-nodevertical-script-cm.yml.j2

```diff
@@ -143,3 +168,121 @@ data:
           privileged: false
       nodeSelector:
         nodevertical: 'true'
+  conprof.yaml.template: |
+    scrape_configs:
+    - job_name: 'apiserver0'
+      scrape_interval: 30s
+      scrape_timeout: 10m
+      scheme: https
+      tls_config:
+        insecure_skip_verify: true
+      static_configs:
+      - targets: ['apiserver0-openshift-kube-apiserver.apps.{{clustername}}.perf-testing.devcluster.openshift.com']
```

Comment:
We might specify multiple targets here to avoid repeating ourselves, i.e
```
- targets: ['apiserver0-openshift-kube-apiserver.apps.{{clustername}}.perf-testing.devcluster.openshift.com', 'apiserver1-openshift-kube-apiserver.apps.{{clustername}}.perf-testing.devcluster.openshift.com', 'apiserver2-openshift-kube-apiserver.apps.{{clustername}}.perf-testing.devcluster.openshift.com'] 
```

In this case we should also make the the job_name more generic with something like apiserver .

Reply:
its actually better to avoid specifying multiple targets, as we can then use specific labels for each in this case i'm trying to apply label instance=node_name ( might not happen in this pr but maybe subsequent ones ) and then also add a label called apiserver=true. Having more detailed labels will help us easily get results, vs trying to figure out which profile is associated with which master.